### PR TITLE
New package: LookingGlass-B1

### PR DESCRIPTION
**Pull request by bra1nwave**
closes https://github.com/void-linux/void-packages/issues/13372

### CODE REVIEW

File: srcpkgs/LookingGlass/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ libX11-devel nettle-devel spice-devel"
+short_desc="Low latency KVMFR implementation for guests with VGA PCI Passthrough"
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/gnif/LookingGlass"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=fa9b3b380f451551c8b40b56a3e873607aa066e8333b7baacd0ed21a05bbb5c5
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64-musl) configure_args+=" -DENABLE_BACKTRACE=OFF" ;;
```
Comment (Duncaen) [inline srcpkgs/LookingGlass/template]: this looks like it should be `*-musl`.

**Comment by Chocimier** [comment]
Usually we do not accept beta software to repository, so this will need to wait for stable release.

**Comment by ericonr** [comment]
Release history shows release candidates for beta releases https://github.com/gnif/LookingGlass/releases ; perhaps upstream can make their stance on releases clearer in order to help packaging effots.

**Comment by ericonr** [comment]
B5 is an actual release, apparently.

**Comment by dm17** [comment]
Would be great to have this package! Thanks :)

**Comment by gnif** [comment]
> B5 is an actual release, apparently.

It's still Beta, it's not a final release. Please do not package this project.

**Comment by gnif** [comment]
> > It's still Beta, it's not a final release. Please do not package this project.
> 
> You don't want distros to package your project? That is surprising... It would get more people using it - and it isn't something you'd use by accident.

Of-course we do eventually. At the moment we do not as you have to keep the Windows app and the client app versions in sync as the protocol they use to communicate is still in flux. Each time a distro has packaged our application we get inundated with support requests because of miss-matched versions.

Until LG reaches version 1.0 when we set the protocol and feature set in stone, LG is not ready for packaging.

**Comment by dm17** [comment]
Ah that makes sense. Thanks.

**Pull request closed without merging**